Update run-a-message-trace-and-view-results.md (#2852)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeOnline/monitoring/trace-an-email-message/run-a-message-trace-and-view-results.md

  - **TRANSFER**: Recipients were moved to a bifurcated message because of content conversion, message recipient limits, or agents.
  - **DEFER**: The message delivery was postponed and may be re-attempted later.
  - **RESOLVED**: The message was redirected to a new recipient address based on an Active Directory look up. When this happens, the original recipient address is listed in a separate row in the message trace along with the final delivery status for the message.
  - **DLP rule**: The message had a DLP rule match in this message.

>   - **Sensitivity label:** A server side labeling event occurred. For example, a label automatically added to a message that includes an action to encrypt or added via the web or mobile client. This action is completed by the Exchange server and logged. A label added via Outlook will not be included in the event field.

Reviewer: ```suggestion
  - **Sensitivity label:** A server-side labeling event occurred, such as a label automatically added to a message that includes an action to encrypt or added via the web or mobile client. This action is completed by the Exchange server and logged. A label added via Outlook will not be included in the event field.
```
